File: SDKs/superset.apache.local/superset/v1/python/src/sdk/models/shared/datasetrelatedcharts.py
from dataclasses import dataclass, field
from typing import List,Optional
from dataclasses_json import dataclass_json
from . import datasetrelatedchart


@dataclass_json
@dataclass
class DatasetRelatedCharts:
    count: Optional[int] = field(default=None, metadata={'dataclasses_json': { 'field_name': 'count' }})
    result: Optional[List[datasetrelatedchart.DatasetRelatedChart]] = field(default=None, metadata={'dataclasses_json': { 'field_name': 'result' }})
    

File: SDKs/superset.apache.local/superset/v1/typescript/src/sdk/models/shared/datasetrelatedcharts.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import { DatasetRelatedChart } from "./datasetrelatedchart";



export class DatasetRelatedCharts extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=count" })
  count?: number;

  @SpeakeasyMetadata({ data: "json, name=result", elemType: DatasetRelatedChart })
  result?: DatasetRelatedChart[];
}
